processing-service-core/api/models/db/globalConfigStore.js

```diff
@@ -91,10 +91,19 @@ function getGlobalConfig(key) {
             }
             // the object returned can have additional properties specific to the database model, only return properties that
             // caller is interested in rather than letting these DB implementation properties leak out of this layer
+            var retrievedScope;
+            switch(findResult.scope){
+                case 1: 
+                    retrievedScope = "REPOSITORY";
+                    break;
+                default:
```

Comment:
Surely this should be `case 0:` rather than `default`.  The default branch should throw an exception as invalid data has been returned from the database.